tests/test_endpoints.py

```diff
@@ -1,24 +1,84 @@
 from requests_flask_adapter import Session
-from kirby.web import app_maker
 from pytest import fixture
+from datetime import datetime
+
+from kirby.web import app_maker
+from kirby.models import db, Environment, Job, JobType, Context, Script, Topic
 
 BASE_API = "http://some-test-server.somewhere"
 
 
+@fixture(scope="function")
+def webapp():
+    config = {
+        "TESTING": True,
+        "SQLALCHEMY_DATABASE_URI": "sqlite://",
+        "SQLALCHEMY_TRACK_MODIFICATIONS": False,
+    }
+    app = app_maker(config=config)
+    with app.app_context():
+        db.create_all()
+        yield app
+
+
 @fixture
-def session():
-    app = app_maker()
-    Session.register(BASE_API, app)
+def session(webapp):
+    Session.register(BASE_API, webapp)
     return Session()
 
 
-def test_it_register_a_job(session):
-    result = session.post(
+@fixture
+def db_populated():
+    env = Environment(name="test")
+    job = Job(
+        name="Booking",
+        description="This is a process for booking orders.",
+        type=JobType.TRIGGERED,
+    )
+    context = Context(environment=env, job=job)
+    script = Script(
+        package_name="booking_process",
+        package_version="1.0.1",
+        context=context,
+        first_seen=datetime.utcnow(),
+    )
+    orders_topic = Topic(name="Orders")
+    asset_management_topic = Topic(name="AssetManagement")
+
+    db.session.add_all(
+        [env, job, context, script, orders_topic, asset_management_topic]
+    )
+    db.session.commit()
+
+
+def get_id_topic(session, name):
```

Comment:
wouldn't that be `get_topic_id` instead?

Reply:
The function is now renamed